internals-js/src/operations.ts

```diff
@@ -1400,7 +1400,7 @@ export class NamedFragments {
   //   enough of a niche case that we ignore it. Note in particular that one sub-case of this rule that is likely
   //   to be common is when the subset ends up being just `__typename`: this would basically mean the fragment
   //   don't really apply to the subgraph, and that this will ensure this is the case.
-  private selectionSetIsWorthUsing(selectionSet: SelectionSet): boolean {
+  static selectionSetIsWorthUsing(selectionSet: SelectionSet): boolean {
```

Comment:
Let me see. Do you have a suggestion where to put it?

Reply:
I can place it just above the `export class NamedFragmentDefinition ...` line.

Reply:
I just pushed an update.
- I moved the function below the `class SelectionSet`, since it's about that class.
- I changed the name to `selectionSetIsWorthReusing` so it makes more sense outside its original context.